### PR TITLE
Adds Hotline Miami-inspired items and devices

### PR DESCRIPTION
**Pull request by ghost**
**Do you like hurting other people?**
If the answer to this question is yes, put a thumbs-up emoji in the comments below.

This PR adds three things:
1. Weapons inspired by Hotline Miami; only two are obtainable normally for now, the others are admin-spawn
2. A voice synthesizer which speaks what you enter into it; basically lets you talk while disguised. Only obtainable through traitor bundle at the moment
3. A new traitor bundle: the Jacket bundle. Included is a voice synthesizer, chicken mask, letterman jacket, classic jeans, either a knife or machete, revolver, and noslips

Weapon sprites are from WalterJe's sprite thread, with two-handed sprites made by me. The voice synthesizer tape sprite was made by me; after a suggestion I edited the universal recorder sprite to make it.

For the record, I wanted to make custom masks but I'm terrible at spriting and I didn't have the patience for trying to make them look at least semi-decent. If sprites are provided, I _will_ add animal masks.

Weapon descriptions allude to the names of songs in the soundtrack of Hotline Miami (Hydrogen, Paris, Inner Animal, etc.)

Below is the in-hand and object sprites for everything.
weapons

### CODE REVIEW

**Comment by Iamgoofball** [comment]
you forgot the commit names

**Comment by Xhuis** [comment]
Ok there should be no direct references this time, if there is, it's an oversight and I'll fix it ASAP if you tell me whe

**Comment by Xhuis** [comment]
No I didn't though.
Hotline Miami-inspired
Inspired
It's also in the changelog so I don't see how it would be worse here

**Comment by Iamgoofball** [comment]
we're ready to hurt people/10

**Comment by dannno** [comment]
richard.dm is a poor name imo

**Comment by Xhuis** [comment]
it's a ___reference___
this entire pr is a reference
get it together man

**Comment by dannno** [comment]
poor excuse

**Comment by Xhuis** [comment]
what do you propose

File: code/game/objects/items/weapons/richard.dm [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/obj/item/weapon/psycho
+	name = "MEME KNIFE"
+	desc = "Yell at Xhuis if you see this!"
```
Comment (ike709) [inline code/game/objects/items/weapons/richard.dm]: @Xhuis YELL

**Comment by Nienhaus** [comment]
Why not put the sprites in where the other object sprites are instead of a whole new dmi?

**Comment by Xhuis** [comment]
I didn't know where to put them, it was more organized that way, and it made them easier to work with

**Comment by Nienhaus** [comment]
For the weapon code.
https://github.com/tgstation/-tg-station/blob/master/code/game/objects/items/weapons/melee/misc.dm
For the sprites.
https://github.com/tgstation/-tg-station/blob/master/icons/obj/weapons.dmi

**Comment by Xhuis** [comment]
Sounds good, lemme do that now

**Comment by Nienhaus** [comment]
Sounds good, don't forget the map merger.

**Comment by Xhuis** [comment]
OP is updated with more pictures

**Comment by PKPenguin321** [comment]
voice synthesizer sounds actually really useful outside of the bundle, i'd be totally okay with it being in the standard traitor PDA or something

**Comment by Xhuis** [comment]
I don't see many uses for it in the normal uplink, to be honest. Generally, if you've gone through the trouble of concealing your identity, you aren't in any situation to speak anyway, plus someone can see it and immediately attack you for having contraband.

I was considering making it a protolathe construction.

**Comment by Lumipharon** [comment]
Why is the machete only 12 force but the baseball bat is 17? (17 is pretty robust)
Also actual baseball when? I just want to have a baseball team that totally aren't cultists/ops/revs/gangsters.

**Comment by Xhuis** [comment]
The machete can be thrown and embeds
The baseball bat is much larger and heavier
It's mainly for balance

**Comment by RandomMarine** [comment]
>The machete can be thrown and embeds.
You sure about that?
https://github.com/tgstation/-tg-station/blob/master/code/__DEFINES/combat.dm#L68

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -101,4 +101,119 @@
...
+	return
+
+/obj/item/weapon/psycho/machete
+	name = "machete"
+	desc = "A utilitarian machete. Bring out your inner animal."
+	icon_state = "machete"
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	force = 12
+	throwforce = 14
+	embed_chance = 33
```
Comment (Xhuis) [inline code/game/objects/items/weapons/melee/misc.dm]: yep

**Comment by Xhuis** [comment]
wait i forgot the sharp flags, so no
let me do that

**Comment by Xhuis** [comment]
or not

**Comment by Xhuis** [comment]
ok i'm not sure how that works, i'll sort it out once i finish coding shitty suggestion implementation

**Comment by Lumipharon** [comment]
throw_speed = 4
and
sharp

Pretty sure having those two will let it embed. While you're at it, pretty sure throwing a hatchet currenly cant embed, which I think is pretty dumb.

**Comment by RemieRichards** [comment]
Not entirely sure this if this is fine under reference policy.

For embedding you need to add the item to either is_sharp() or can_embed(), either will work. (can_embed() reads from is_sharp())

**Comment by optimumtact** [comment]
The reference policy is no player references

There is no copyright risk here

I see no legal or policy reason to avoid these changes

File: code/game/objects/items/weapons/melee/misc.dm [hunk 3]
```diff
@@ -101,4 +101,119 @@
...
 	playsound(src.loc, 'sound/weapons/batonextend.ogg', 50, 1)
-	add_fingerprint(user)
\ No newline at end of file
+	add_fingerprint(user)
+
+
+//Hotline Miami-inspired weapons
+/obj/item/weapon/psycho
+	name = "MEME KNIFE"
+	desc = "Yell at Xhuis if you see this!"
```
Comment (optimumtact) [inline code/game/objects/items/weapons/melee/misc.dm]: I take it back, this is probably a violation

no self insert items

**Comment by killerfgs** [comment]
Made a Jacket.. Jacket sprite for funsies, could mebbe replace the bomber jacket in the bundle http://www.mediafire.com/view/tobvaay45amscsq/Jacketv2.dmi

**Comment by Chocobro** [comment]
I'M A VERY BIG FAN OF ALL OF THESE SPRITES
you've outdone yourself xhuis
shoutouts to the chicken head that's the best

File: code/game/objects/items/weapons/melee/misc.dm [hunk 4]
```diff
@@ -101,4 +101,119 @@
 		attack_verb = list("hit", "poked")
 
 	playsound(src.loc, 'sound/weapons/batonextend.ogg', 50, 1)
-	add_fingerprint(user)
\ No newline at end of file
+	add_fingerprint(user)
+
+
+//Hotline Miami-inspired weapons
+/obj/item/weapon/psycho
```
Comment (Miauw62) [inline code/game/objects/items/weapons/melee/misc.dm]: remove this comment
why do these weapons need a separate subtype? add them under weapon/melee

**Comment by Miauw62** [comment]
this is :+1: apart from this minor nitpick.

**Comment by Xhuis** [comment]
Will remove the comment. I made them a subtype so I didn't have to put a new 'icon =' thing for every weapon. I'll do some more stuff in the next update though.

**Comment by Supermichael777** [comment]
i would prefer if the voice changer tape recorder shared the tape recorder sprite we already have.
and if you got a throw-able stun knife instead of a frying pan. but still cool
oh and don't forget to add the mask to the barnyard curse

**Comment by Xhuis** [comment]
I didn't make the mask. It's in the game by default from the AutoDrobe.

**Comment by Nienhaus** [comment]
You could modify the universal recorder a bit.

**Comment by Xhuis** [comment]
>merge conflicts
pls

**Comment by Xhuis** [comment]
squashed commits
still have no idea why this is happening because i fixed the files in question but w/e i'll work on it

**Comment by Xhuis** [comment]
yay

**Comment by pudl** [comment]
remove the medkit from the jacket box, that makes it pretty much 100% more op. otherwise it's fine

**Comment by Iamgoofball** [comment]
@pudl it's a traitor bundle, you drop all your telecrystals on it for an rng chance to get something extremely shitty

I say leave the medkit

**Comment by pudl** [comment]
@iamgoofball when compared to other kits, this one is basically the roll to hope for. a good weapon, memetic disguise, voice synth, and a TRAITOR medkit (that means it comes with a night hud, a syringe of chloral, a hypo, and a defib). if you want it to be good for healing, then just give them a normal medkit. you could make the tactical medkit by itself a possible roll and that would STILL be op. under normal circumstances, only ops should be getting one at all.

**Comment by Lumipharon** [comment]
the hypo (assuming you can refill it) is EXTREMELY powerful, in the right hands (ie: Anyone that can get their hands on a chem dispenser.

**Comment by Xhuis** [comment]
I'll probably change the medkit. I was strongly considering it anyway, as you can see in the op

**Comment by PKPenguin321** [comment]
Medkit is the only thing that makes this bundle really worth it as-is considering there's not a lot of traitor-y items in it to begin with.
Maybe put an adrenal implant and some stimulants in there, if the medkit's gonna go?

**Comment by Xhuis** [comment]
why the FUCK is this still conflicting

**Comment by Lumipharon** [comment]
adrenal implant alone is pretty good, shit stun breaks, makes you go fast and heals.

**Comment by Xhuis** [comment]
replaced noslips with adrenal implant
shitty commit text is because i was going to squash commits but i can't b/c i merged all the updates into it

**Comment by Xhuis** [comment]
ok there's so much shit with this branch going on
people keep merging updates that got pr'd after what i made
i'm just gonna fix this when i get the time ok

**Pull request closed without merging**